### PR TITLE
returning the raw response body string if it does not start with quotes

### PR DESCRIPTION
**Pull request by rassmate**

### CODE REVIEW

**Review by JonasHallFnx — APPROVED** [review]

**Review by magnusp — APPROVED** [review]

**Review by jepp3 — APPROVED** [review]

File: client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java [hunk 1]
```diff
@@ -528,6 +530,10 @@ private void setHeaderParams(RequestBuilder request, Method method, Object[] arg
             return Mono.empty();
         }
 
+        if (String.class.equals(type) && !string.startsWith(QOUTES) && !"null".equalsIgnoreCase(string)) {
```
Comment (jepp3) [inline client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java:533]: qoutes -> quotes

Comment (jepp3) [reply client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java:533]: maybe im thinking wrong here. but should it not be: 

if( is String and starts with quote ) ?

Comment (splitfeed) [reply client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java:533]: > qoutes -> quotes

I actually think it should be just quote, since it only contains one quotation mark  :)

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java [hunk 2]
```diff
@@ -961,6 +962,18 @@ public void shouldDeserializeVoidResult() {
         server.disposeNow();
     }
 
+    @Test
+    public void shouldDeserializeSimpleStringWithoutJsonQuotes() {
```
Comment (splitfeed) [inline client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java:966]: Seems like you are missing a test case for when the response is `"null"` . Are there tests that cover strings that are quoted, so that is validated to be working?

**Review by splitfeed — CHANGES_REQUESTED** [review]

File: client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java (hunk 1 above)
Comment (rassmate) [reply client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java:533]: If the string starts with quotes we need to run it through the objectMapper

Comment (rassmate) [reply client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java:533]: I will change spelling and remove an S

The null test is already implemented in RestClientFactoryTest

File: client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java (hunk 2 above)
Comment (rassmate) [reply client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java:966]: The "null" test is implemented in RestClientFactoryTest.shouldDeserializeNullToEmpty()

**Review by jepp3 — APPROVED** [review]

**Review by splitfeed — APPROVED** [review]

**Review by pontusjo — APPROVED** [review]
